### PR TITLE
doubly link list

### PR DESCRIPTION
**Pull request by TheSTL**
#### Add doubly link list  data structure having following functions:
- addAtBeginning
- addAtEnd
- removeAtBeginning
- removeAtEnd
- remove `argument node`
- length
- display

#### Add LRU-cache data structure implementation.
- get(key) `O(1)`
- set(key, value) `O(1)`

### CODE REVIEW

**Comment by ashokdey** [comment]
Great move @TheSTL 👍 

The first thing I want you to do is to remove the LRU implementation in a folder named `_algorithms_`. Meanwhile, I am reviewing the DLL implementation

File: src/_DataStructures_/DoublyLinkedList/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  length() {
+    return this.size;
+  }
+
+  display() {
+    let address = this.head.next;
+    while (address !== this.tail) {
+      console.log(address.data);
```
Comment (ashokdey) [inline src/_DataStructures_/DoublyLinkedList/index.js]: **no-console** 

Also, make sure the code adheres to ESLint rules.

**Comment by TheSTL** [comment]
@ashokdey  
Changes are done.

**Comment by ashokdey** [comment]
You forgot to add entries in the README file! 🤒

File: src/_DataStructures_/DoublyLinkedList/index.js (hunk 1 above)
Comment (TheSTL) [reply src/_DataStructures_/DoublyLinkedList/index.js]: It's a display function to print data in a linked list.

Comment (ashokdey) [reply src/_DataStructures_/DoublyLinkedList/index.js]: Add the comment to ignore the ESLint rule

**Pull request merged**